ibis/backends/tests/test_timecontext.py

```diff
@@ -75,3 +75,31 @@ def test_context_adjustment_filter_before_window(alltypes, df, context):
         expected = expected.reset_index(drop=True)
 
         tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.only_on_backends(['pandas', 'pyspark'])
+def test_grouped_bounded_expanding_window(
```

Comment:
How does this test cover your change? Isn't the context not adjusted for expanding window?

Reply:
Discussed offline. 

@LeeTZ Can you verify that existing context adjustment tests fail without the change in selection.py?